### DISCUSSION TITLE
Disable redirect on error

### DISCUSSION THREAD

**Discussion by callumok2004** [nextauthjs/next-auth#2692] [Help] [answered]
I want to use the `signIn` function to create a custom login experience for the user, but when the credentials are incorrect, it refreshes the page, how can I have it show the error in the callback of the function without the page refreshing and handle the error myself?

**Comment by callumok2004** [comment, marked as answer]
Nevermind, I completely misread the docs :D

**Reply by kcrtr** [reply]
Can you show me where in the docs you found the solution?
